utils/check_dummies.py

```diff
@@ -156,8 +160,11 @@ def check_dummies(overwrite=False):
 
     actual_dummies = {}
     for backend, file_path in dummy_file_paths.items():
-        with open(file_path, "r", encoding="utf-8", newline="\n") as f:
-            actual_dummies[backend] = f.read()
+        if os.path.isfile(file_path):
+            with open(file_path, "r", encoding="utf-8", newline="\n") as f:
+                actual_dummies[backend] = f.read()
+        else:
+            actual_dummies[backend] = ""
```

Comment:
This removes the error that appeared when running `make fix-copies` and the corresponding dummy files did not exist yet.